### PR TITLE
docs for tooltips

### PR DESCRIPTION
**Pull request by AnthonySteele**
Docs for tooltips on the StatsDConfiguration.
These should come up in intellisense when using this object.

### CODE REVIEW

File: src/JustEat.StatsD/StatsDConfiguration.cs [hunk 1]
```diff
@@ -8,13 +8,35 @@ public class StatsDConfiguration
...
         public static readonly TimeSpan DefaultDnsLookupInterval = TimeSpan.FromMinutes(5);
 
+        /// <summary>
+        /// The host name or IP address of the statsD server. 
+        /// This field must be set. 
+        /// </summary>
         public string Host { get; set; }
 
+        /// <summary>
+        /// The port on the satsD server that stas are sent to.
```
Comment (martincostello) [inline src/JustEat.StatsD/StatsDConfiguration.cs]: Typo: "stas"

File: src/JustEat.StatsD/StatsDConfiguration.cs [hunk 2]
```diff
@@ -8,13 +8,35 @@ public class StatsDConfiguration
...
+        /// Default is the standard port (8125).
+        /// </summary>
         public int Port { get; set; } = DefaultPort;
 
+        /// <summary>
+        /// Length of time to cache the host name to Ip address lookup.
```
Comment (martincostello) [inline src/JustEat.StatsD/StatsDConfiguration.cs]: Use "IP".

**Comment by slang25** [comment]
I'm a little late to the party, but I think your last PR isn't as thread-safe as it previously was (or should be). I'll take another look when I'm more awake =)

**Comment by AnthonySteele** [comment]
@slang25 It's likely, I haven't given a lot of thought to that. Are you talking about concurrency in the `CachedIpEndpointSource`, or something else?

**Comment by AnthonySteele** [comment]
Merge this one? It should be good.

**Pull request merged**